Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/MSBuildTemplates/SDKProjectTemplate.g.props.template

```diff
@@ -9,9 +9,8 @@
     <ProjectGuid><!--PROJECT_GUID_TOKEN--></ProjectGuid>
     <AllowUnsafeBlocks><!--ALLOW_UNSAFE_TOKEN--></AllowUnsafeBlocks>
     <LangVersion><!--LANGUAGE_VERSION_TOKEN--></LangVersion>
-    <DevelopmentBuild><!--DEVELOPMENT_BUILD_TOKEN--></DevelopmentBuild>
+    <DevelopmentBuild>false</DevelopmentBuild>
```

Comment:
Intentionally hardcoded here?

Reply:
Yes, as several other properties. I never actually set this to anything other than false, and override it in certain conditions in this project.